test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -39,7 +39,7 @@ declare module 'test-case-reporting' {
   /** A job within a Travis build. */
   export interface Job {
     jobNumber: string,
-    type: JobType,
+    testSuiteType: TestSuiteType, // e.g. unit, integration, etc.
```

Comment:
Here you're referencing the type, so it's obvious. Below it's just a string, which is why I suggested the comment

Reply:
Removed.